mula/scheduler/models/tasks.py

```diff
@@ -82,6 +132,13 @@ class TaskDB(Base):
         default=TaskStatus.PENDING,
     )
 
+    pending = Column(Interval)
+    queued = Column(Interval)
+    dispatched = Column(Interval)
+    running = Column(Interval)
+
+    meta = Column(JSONB)
```

Comment:
Added for runners to relay and persist information about a task, e.g. cpu, mem load, run times etc. If this info is already available through bytes we can opt to reference that. However, it might be more difficult/time-consuming to retrieve that information when we want to make rankings based on that information since we don't have it readily available in the scheduler.

Reply:
We can opt for removing this field for now since it depends on changes or implementation in runners

Reply:
I'd rather collect this info in the scheduler for fast-access, as its something the user wants to filter on, see on the tasklist. Having those details only in Bytes would mean we end up reading those jobmeta's for the whole list / or worse for all jobs when rendering a filtered task list.

Reply:
The question where and how we want to to store these meta values in the schedulers database is still very valid though.